docs/notebooks/expected_improvement.pct.py

```diff
@@ -174,11 +167,11 @@ def build_model(data):
 # We can also inspect the model hyperparameters, and use the history to see how the length scales evolved over iterations
 
 # %%
-gpflow.utilities.print_summary(model[OBJECTIVE].model)
+gpflow.utilities.print_summary(result.try_final_models()[OBJECTIVE].model)
 
 ls_list = [
     step.models[OBJECTIVE].model.kernel.lengthscales.numpy()  # type: ignore
-    for step in result.history
+    for step in result.history + [result.final_result.unwrap()]
```

Comment:
this relates to my comment on the OptimizationResult class - would be nice to explicitly clarify that result.history[-1] != result.final_result (if it does exist)

Reply:
done